README.md

```diff
@@ -1,8 +1,48 @@
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Python, Maven, AWS, C++, C, JavaScript, HTML/CSS, Android, Docker, JSON
+
+**Familiar:** Flask, Google Cloud Platform, Blockchain, Heroku, UML, PHP
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui/tree/master/linux_sql)]: Developed an application with Bash, Docker, Git, and PostgreSQL to record the hardware specifications and monitor resources usages (CPU/memory, etc.) of nodes within a Linux cluster. The data collected will be stored in a PostgreSQL database and used to generate reports for resource planning with SQL queries. Crontab was effectively used to collect continuous resource usage data from a cluster, which was later used to generate the report.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui/tree/master/core_java)]: Grep App. Designed an application that mimics the Linux grep command with Core Java. The purpose of this application is to search through a given directory for any matches to a user-given string and write the found matching strings to an output file. The technologies used include Java 8, IntelliJ IDE, Maven, Docker, and Git.
+
+
+## Highlighted Projects
+**IoT Smart Home Security System**: Led a team consisting of four Software Engineering students to develop a Smart Home Security System using Arduino, Java, C, ESP8266 Wi-Fi Module, PIR Motion Sensor, Android, Firebase, and ArduCAM Mini. Application offers arm/disarm feature, voice command capability via Amazon Alexa, live video camera footage and capability to record footage. The application was built using the agile design methodology and  tested using the big bang integration testing method. This was a final capstone project and some responsibilities included but were not limited to weekly reports describing work done in the week, weekly meetings with faculty advisor, meetings with group members three times a week, and prototype demonostration.
```

Comment:
**IoT Smart Home Security System**: Led a team consisting of four Software Engineering students to develop a Smart Home Security System using Arduino, Java, C, ESP8266 Wi-Fi Module, PIR Motion Sensor, Android, Firebase, and ArduCAM Mini. The application offers an arm/disarm feature, voice command capability via Amazon Alexa, live video camera footage, and record footage. The application was built using the Agile design methodology and tested using the big bang integration testing method. This was a final capstone project, and some responsibilities included but were not limited to weekly reports describing work done in the week, weekly meetings with the faculty advisor, meetings with group members three times a week, and prototype demonstration.